src/sigstore.ts

```diff
@@ -44,41 +44,41 @@ type IdentityProviderOptions = Pick<
 export async function sign(
   payload: Buffer,
   options: SignOptions = {}
-): Promise<Bundle> {
+): Promise<string> {
```

Comment:
I was going back and forward on this change, initially I thought that maybe we should expose a bundle serialiser/deserialiser method on the `sigstore` export but exposing the `Bundle` interface didn't feel quite right as the format includes implementation details from the proto <> typescript conversion.

But then again, if you used this lib from typescript it would be nice to get a typed object back! Thoughts on this?

Reply:
also cc @kommendorkapten as sigstore bundle implementor

Reply:
While I'm not a JS/TS expert, I do feel that the sigstore-js library should acept/produce bundle objects, as that's what's native to the APIs? I think internally sigstore-js should not perform any operations on transforming data more than really necessarily, if client's prefer to use a diferent object model, I think then clients can implement that wrapper layer as that would be a domain logic decision?

Reply:
> sigstore-js library should acept/produce bundle objects

You mean the json representation on disk or a typed bundle object? Currently the API requires the caller to transform the json representation into a `Bundle` interface by calling the `Bundle.fromJSON` method, which is internal to the library and generated from protobufs in cosign.

Reply:
For JS/TS I would have to let @bdehamer answer what's the common naming schema when using protoserializers. But  I'm not sure that sigstore-js itself should expose those methods? I would think that marshalling is not something that sigstore-js is responsible for, but a separate package exists, that defines the data objects, and any serialization methods. 

The approach I (and some others) are to create a new repo in sigstore, `sigstore/protobuf` where the actual protobuf files exist, and also it's from there where all the language clients are built, and published to the appropriate repository. Then sigstore-js can import the package from there.

Reply:
> Then sigstore-js can import the package from there.

Hm this could be a good longer term aim but we'll need to figure something out to be able to use this library from npm land. Currently blocked on this.

@bdehamer thoughts on exposing some methods to serialize/deserialize the bundle so we can use it with the json bundle file?